requirements/edx/base.txt

```diff
@@ -69,7 +69,7 @@ django-model-utils==3.0.0
 django-mptt==0.8.7
 django-multi-email-field==0.5.1  # via edx-enterprise
 django-mysql==2.4.1
-git+https://github.com/edx/django-oauth-plus.git@01ec2a161dfc3465f9d35b9211ae790177418316#egg=django-oauth-plus==2.2.9.edx-1
+git+https://github.com/edx/django-oauth-plus.git@01ec2a161dfc3465f9d35b9211ae790177418316#egg=django-oauth-plus==2.2.9.edx-2
```

Comment:
updated the `django-oauth-plus` version here

Reply:
@feanil do we need to update hash here also or version update is enough ?

Reply:
@awais786  yes we need to update the hash to use the latest release hash. 
Also remove this from base and move this to github.in

Reply:
remove the changes from base as you have updated github.in